docs/content/en/docs/packages/best_practice.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Configuration best practices"
 linkTitle: "Best practices"
-weight: 15
+weight: 6
```

Comment:
This was not great, it is much better to leave gaps in the weight, so we aren't renumbering *everything* on a page shuffle.